logins-api/android/LoginsSampleApp/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
+        classpath 'com.android.tools.build:gradle:3.1.4'
```

Comment:
eh?  Automatic from Android Studio?

Reply:
Yes, sorry. didn't intend to change this.